plugins/chat/assets/stylesheets/common/chat-message-thread-indicator.scss

```diff
@@ -0,0 +1,29 @@
+.chat-message-thread-indicator {
```

Comment:
Chapoi can refine these styles as we go, don't want to hold up PRs just for this.

Reply:
IMO the new style is too subtle and doesn't leave room for the excerpt or the avatars of the people in the thread, but yeah I guess she can bring it back / make more changes later